apps/spruce/src/components/DayPicker/DayPicker.test.tsx

```diff
@@ -75,4 +75,16 @@ describe("daypicker", () => {
       false,
     ]);
   });
+
+  it("disables widget", async () => {
+    const onChange = jest.fn();
+    render(<DayPicker disabled onChange={onChange} />);
```

Comment:
nit. Seems like it'd be okay to inline the function since it isn't being used
```suggestion
    render(<DayPicker disabled onChange={jest.fn()} />);
```